### PR TITLE
feat(combo-card): added combochart card

### LINKED ISSUE #1483: [Charting] Combo charts - area and line
### Summary

Carbon Charts have been working on the ability to overlay 2 different charts on top of each other, and we want to leverage that capability to show an area chart with one value and show a line chart with other values on top of it.

**Additional context**
Issue in chart repo: https://github.com/carbon-design-system/carbon-charts/issues/527
This issue indicates that it should have been delivered in 2Q2020

Proposed design that needs it

Health score is the area chart and drivers are the line chart another.  They share the same Y axis values.  Additionally, we would like to overlay an event timeline on the same X axis, but I'm not sure if Carbon Charts has that option available yet.

### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?

This chart is scheduled for Health / MAS October release.

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

### Available extra resources

What resources do you have to assist this effort?

Predict team will be working on this chart.  It's possible that @navjeets or others might be able to contribute this to IoT.

_Carbon add-ons for Watson IoT is a collaborative system. We encourage teams to build components and submit them for integration._

### PR DESCRIPTION
**Pull request by marcJV**
Closes #1483

**Summary**

- Adds a card wrapper for the Combo Chart

**Change List (commits, features, bugs, etc)**

- Added ComboCardChart component, stories, and tests

**Acceptance Test (how to verify the PR)**

Verify that the chart is rendered correctly at:
https://deploy-preview-1939--carbon-addons-iot-react.netlify.app?path=/story/watson-iot-combochartcard--combo-chart-card-area

Loading state is rendered correctly at:
https://deploy-preview-1939--carbon-addons-iot-react.netlify.app?path=/story/watson-iot-combochartcard--combo-chart-card-loading

Empty state is rendered correctly at:
https://deploy-preview-1939--carbon-addons-iot-react.netlify.app?path=/story/watson-iot-combochartcard--combo-chart-card-empty

### CODE REVIEW

File: src/components/ComboChartCard/ComboChartCard.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <Card
+      isEmpty={isEmpty(values)}
+      className={classnames(className, `${iotPrefix}--combo-chart-card`)}
+      title={`${title === '' ? chartTitle : title}`}
+      {...others}>
+      <div className={`${iotPrefix}--combo-chart-card__container`}>
+        <ComboChart
+          key="combo-chart"
+          data={values}
```
Comment (sstone2423) [inline src/components/ComboChartCard/ComboChartCard.jsx:41]: We do a lot of data formatting in BarChartCard and TimeSeriesCard as well as formatting the tooltip to fit our needs. I think this component will need the same util logic

**Review by sstone2423 — CHANGES_REQUESTED** [review]

File: src/components/ComboChartCard/ComboChartCard.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { CARD_SIZES } from '../../constants/LayoutConstants';
+import Card from '../Card/Card';
+import { CardPropTypes } from '../../constants/CardPropTypes';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  i18n: PropTypes.shape({
```
Comment (sstone2423) [inline src/components/ComboChartCard/ComboChartCard.jsx:16]: We'll need the same i18n as TimeSeriesCard and BarChartCard I think as we translate as many things as possible

**Review by sstone2423 — CHANGES_REQUESTED** [review]

**Review by sstone2423 — CHANGES_REQUESTED** [review]
There should also be an expand option that renders a table with all of the data being rendered. See the other chart cards for a reference

**Comment by davidicus** [comment]
Opened #1976 to track bug with storyshot imports

**Comment by davidicus** [comment]
closing. Dev will reopen in another PR since substantial changes have been made since first reviewed.

**Pull request closed without merging**